src/components/ComboChart.tsx

```diff
@@ -14,13 +14,14 @@ import { convertBucketsToAFM, convertBucketsToMdObject } from "../helpers/conver
 import { getResultSpec } from "../helpers/resultSpec";
 import { MEASURES, SECONDARY_MEASURES, VIEW } from "../constants/bucketNames";
 import { setMeasuresToSecondaryAxis } from "../helpers/dualAxis";
-import { sanitizeConfig, sanitizeMeasures } from "../helpers/optionalStacking/common";
+import { sanitizeConfig, sanitizeComputeRatioOnMeasures } from "../helpers/optionalStacking/common";
+import { IChartConfig } from "..";
 
 export interface IComboChartBucketProps {
-    columnMeasures?: VisualizationInput.IMeasure[];
-    lineMeasures?: VisualizationInput.IMeasure[];
-    primaryMeasures?: VisualizationInput.IMeasure[];
-    secondaryMeasures?: VisualizationInput.IMeasure[];
+    columnMeasures?: VisualizationInput.AttributeOrMeasure[];
```

Comment:
why this change?

Reply:
To make it consistency with other visualizations (eg Column/Bar....)
And to get rid of typescript error because `sanitizeComputeRatioOnMeasures` takes `BucketItem` as parameter and it covers for Column/Bar/Area charts as well.

Reply:
Reverted